### DISCUSSION TITLE
can not get organisation stats

### DISCUSSION THREAD

**Discussion by sekomer** [anuraghazra/github-readme-stats#2360] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I deployed my own Vercel app to get private repository stats and achieved that. But can not get my stats from my organizations, how can I achieve this? 
thanks

### Describe the solution you'd like

I want to see my organisation stats on my profile too

### Describe alternatives you've considered

I've tried `include_orgs` query parameter on the request as stated by some other people but this doesn't work too.

[ here's my deployment and query ]
https://sexomer.vercel.app/api/top-langs?username=sekomer&include_orgs=true&show_icons=true&locale=en&layout=compactincluding

**Comment by rickstaa** [comment, marked as answer]
@Sekomer Thanks for your question. This is one of our most requested features (see https://github.com/anuraghazra/github-readme-stats/issues/1935). It has already been implemented in https://github.com/anuraghazra/github-readme-stats/pull/2277 but still needs tests. You can, however, already use it (see https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-1329809068).
